src/test/java/io/xstefank/wildfly/bot/PRMentionsReviewsCombinationTest.java

```diff
@@ -115,20 +116,23 @@ public void testMentionsReviewsReviewsHitSameRule() throws IOException {
                 .build();
         mockedContext = MockedContext.builder(gitHubJson.id())
                 .prFiles("src/main/java/resource/application.properties")
-                .collaborators("user1", "user2");
+                .users("user1", "user2");
 
         given().github(mocks -> Util.mockRepo(mocks, wildflyConfigFile, gitHubJson, mockedContext))
                 .when().payloadFromString(gitHubJson.jsonString())
                 .event(GHEvent.PULL_REQUEST)
                 .then().github(mocks -> {
                     Mockito.verify(mocks.pullRequest(gitHubJson.id()), Mockito.never())
                             .comment(ArgumentMatchers.anyString());
+                    List<String> users = new ArrayList<>(List.of("user1", "user2"));
                     ArgumentCaptor<List<GHUser>> captor = ArgumentCaptor.forClass(List.class);
-                    Mockito.verify(mocks.pullRequest(gitHubJson.id())).requestReviewers(captor.capture());
-                    Assertions.assertEquals(captor.getValue().size(), 2);
-                    MatcherAssert.assertThat(captor.getValue().stream()
-                            .map(GHPerson::getLogin)
-                            .toList(), Matchers.containsInAnyOrder("user1", "user2"));
+                    Mockito.verify(mocks.pullRequest(gitHubJson.id()), Mockito.times(2)).requestReviewers(captor.capture());
+                    Assertions.assertEquals(captor.getAllValues().size(), 2);
+                    for (List<GHUser> requestedReviewer : captor.getAllValues()) {
+                        Assertions.assertEquals(1, requestedReviewer.size());
+                        users.remove(requestedReviewer.get(0).getLogin());
+                    }
+                    Assertions.assertEquals(0, users.size());
```

Comment:
At least add a message, because this default will be misleading

Reply:
I think the way of `containsAll` method might be nicer after all (even though it's not used :smile: )